spring-session-data-redis/src/main/java/org/springframework/session/data/redis/RedisOperationsSessionRepository.java

```diff
@@ -588,7 +588,7 @@ private void publishEvent(ApplicationEvent event) {
 	}
 
 	public void setRedisKeyNamespace(String namespace) {
-		this.keyPrefix = DEFAULT_SPRING_SESSION_REDIS_PREFIX + namespace + ":";
+		this.keyPrefix = namespace + ":";
```

Comment:
We need to validate that provided namespace is not `null` or empty.

Reply:
How do you suggest to do that? Throw an Exception?

Reply:
Using `org.springframework.util.Assert#hasText` - I see you addressed that, thanks.